syntax = "proto3";

import "src/protos/Event.proto";

message AnnualInflationConfig {
  double rate = 1;
  Event deflation_event = 2;
  Event inflation_event = 3;
}